cmd/docs.go

```diff
@@ -37,5 +37,13 @@ func docsMain(cmd *cobra.Command, args []string) exitCode {
 	} else {
 		cmdutil.Println("No description provided.")
 	}
+
+	if schema.PermissibleChildren {
+		// Intentionally adds an empty newline after the provided description.
+		cmdutil.Println(`
+If a subdirectory is listed in 'stree' but not visible in your directory then you are likely
```

Comment:
I think we should move this over to `stree` and just add a note somewhere in its output. Or move it to the website.

Reply:
So rather than track this, we'll look at including a note about it in documentation on the `stree` command and the website.